# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ladspa-cmt/disable-sound.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./makefile.orig	2002-12-19 15:52:51.000000000 -0500
+++ ./makefile	2004-02-21 02:08:16.000000000 -0500
@@ -80,12 +80,10 @@
 	@echo ---------------------------------------------
 	@echo First listen to the white noise input signal:
 	@echo ---------------------------------------------
-	play ../../ladspa_sdk/snd/noise.wav
 	@echo -------------------------
 	@echo Compare to plugin output.
 	@echo -------------------------
 	@echo Should be a noise band around 6000Hz, repeated quietly after 1s.
-	play /tmp/test.wav
 
 /tmp/test.wav:	$(PLUGIN_LIB) ../../ladspa_sdk/snd/noise.wav
 	analyseplugin $(PLUGIN_LIB)
